src/main/java/database/data/Developer.java

```diff
@@ -0,0 +1,16 @@
+package database.data;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@RequiredArgsConstructor
+public class Developer {
+    private int id;
```

Comment:
https://stackoverflow.com/questions/7506802/using-wrapper-integer-class-or-int-primitive-in-hibernate-mapping